### ISSUE TITLE
I'm very interesting your job. when do you public your code?

### ISSUE THREAD

**Issue by zhaowenlovehome** [CWanli/RecoNet#1]

**Comment by CWanli** [comment]
I‘m working on my new submission for AAAI, I will release the code after AAAI deadline, thank you for your attention!

**Comment by CWanli** [comment]
I'm sorry for the delay but we have very limited GPUs. I have to focus on my new submission now.

**Comment by zhaowenlovehome** [comment]
can I see the  code of Tensor Generation Module(TGM) and Tensor Reconstruction Module(TRM)

**Comment by zhaowenlovehome** [comment]
我想看看你TGM和TRM模块怎么写的，如果用patch训练的化，我不知道在H和W方向怎么进行卷积

**Comment by Aaromxj** [comment]
I am very interested in your work and hope to see your code soon.

**Comment by WenguoLi** [comment]
Yes, I'm also curious about convolution in width and height, Tensor Generation Module(TGM) and Tensor Reconstruction Module(TRM) are very interesting works.

**Comment by Serge-weihao** [comment]
@CWanli 
what's the computational complexity of your TGM module?
![image](https://user-images.githubusercontent.com/40890323/93436083-e4112f80-f8fc-11ea-8ed5-f046f6340e69.png)

**Comment by InvictusRichard** [comment]
> 我想看看你TGM和TRM模块怎么写的，如果用patch训练的化，我不知道在H和W方向怎么进行卷积

我有同样得问题？看似input size需要固定，那么如何进行多尺度测试呢？
